src/redux/configureStore.js

```diff
@@ -0,0 +1,10 @@
+import Redux from 'react-redux';
```

Comment:
- [ ] Good job setting up the redux store but there are some issues you need to fix, You imported redux methods in the wrong way. If you will access your store in your components the application will throw errors and will not work. Please make sure to fix this issue and import redux methods in the correct way.
 - You can consider this code. `import { createStore, combineReducers } from 'redux';`

Reply:
@thecodechaser Thanks for the details in the review 🙏🏾. I will apply these changes right away.